### PR TITLE
Remove use of `echo -e`

### LINKED ISSUE #25: Unnecessarily environment-specific
Tests are running by calling `exec()` with an option like:

```
echo -e 'Date: Wed, 15 Apr 2020 23:38:05 GMT

{"hello": "world"}' | cat <path> - | <generator> canonicalize --headers "date"
```

This seems unnecessarily environment-specific to achieve the fairly simple goal of writing some input to the process.

### PR DESCRIPTION
**Pull request by Diggsey**
Fixes #25

### CODE REVIEW

**Comment by Diggsey** [comment]
Could remove `cat` too, but this is enough to get it working on windows.

**Comment by aljones15** [comment]
thanks for the PR. @liamdennehy do you want to test this?

File: test/latest/util.js [hunk 1]
```diff
@@ -19,17 +19,19 @@ async function generate(file, options) {
...
-  const result = await runTest(command);
+  const result = await runTest(command, inputStr);
   return result;
 }
 
-function runTest(command) {
+function runTest(command, inputStr) {
   return new Promise((resolve, reject) => {
     const child = exec(command);
+    child.stdin.end(inputStr)
```
Comment (aljones15) [inline test/latest/util.js:34]: Hey thank you for using `child.stdin.end` that might actually be what we are looking for. I am approving this, but there are 2 small eslint errors in `util.js` that could be corrected with `eslint --fix`, but if you are busy we can merge this pending another implementation can prove this PR did not break existing functionality (I'm pretty confident it won't).

File: test/latest/util.js [hunk 2]
```diff
@@ -19,17 +19,19 @@ async function generate(file, options) {
     args += value;
   }
   // this cat filePath - the dash is the last pipe op
+  const inputStr = `${latestDate}\n\n{"hello": "world"}`
```
Comment (aljones15) [inline test/latest/util.js:22]:
```suggestion
  const inputStr = `${latestDate}\n\n{"hello": "world"}`;
```

File: test/latest/util.js (hunk 1 above)
Comment (aljones15) [inline test/latest/util.js:34]:
```suggestion
    child.stdin.end(inputStr);
```

**Comment by Diggsey** [comment]
I made the suggested changes - I'll leave the removal of `cat` to a separate PR as it's slightly more involved.

**Comment by aljones15** [comment]
@msporny I don't have the persmission to merge in this PR, but it is quite helpful and takes this test suite further along. Do we know who has write privileges to the parent repo?

**Pull request merged**